radixdlt/src/integration/java/org/radix/integration/RadixTest.java

```diff
@@ -52,7 +52,10 @@ public static void startRadixTest() throws Exception {
 
 		properties = new RuntimeProperties(runtimeConfigurationJSON, null);
 
+		// Tests need this
 		properties.set("debug.nopow", true);
```

Comment:
Show we maybe use CONSTANTS (Enum...?) for Property keys like this?

Reply:
Yes, probably :).  I'm not going to expand the scope of this task to include that, but cool if you want to raise an issue to address it.